README.md

```diff
@@ -1,9 +1,8 @@
 # 🤔 Introduction :
-Ternoa is a Decentralised, Open source, NFT-centric Layer 1 blockchain that is multi-chain by design and aims to provide a technical stack to build scalable and secure NFTs with native support for advanced features.
```

Comment:
This paragraph must be kept, it introduce properly Ternoa.

Reply:
Done